File: src/Omnia.ProcessManagement.Models/ProcessLibrary/ProcessLibraryRequest.cs
using Omnia.Fx.Models.Language;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ProcessLibrary
{
    public class ProcessLibraryRequest
    {
        public string WebUrl { get; set; }
        public Dictionary<string, List<string>> Filters { get; set; }
        public string SortBy { get; set; }
        public bool SortAsc { get; set; }
        public int PageNum { get; set; }
        public int PageSize { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processlibrary/ProcessLibraryRequest.ts
import { LanguageTag } from '@omnia/fx-models';

export interface ProcessLibraryRequest {
    webUrl: string,
    filters?: { [key: string]: Array<string> },
    sortBy?: string,
    sortAsc?: boolean,
    pageNum: number,
    pageSize: number;
}
